Resources/views/CRUD/base_show_field.html.twig

```diff
@@ -10,7 +10,23 @@ file that was distributed with this source code.
 #}
 
 <th{% if(is_diff|default(false)) %} class="diff"{% endif %}>{% block name %}{{ field_description.label|trans({}, field_description.translationDomain ?: admin.translationDomain) }}{% endblock %}</th>
-<td>{% block field %}{% if field_description.options.safe %}{{ value|raw }}{% else %}{{ value|nl2br }}{% endif %}{% endblock %}</td>
+<td>
+    {%- block field -%}
+        {% spaceless %}
+            {% if field_description.options.collapse is defined %}
+                {% set collapse = field_description.options.collapse %}
+                <div class="sonata-readmore"
+                      data-readmore-height="{{ collapse.height|default(40) }}"
+                      data-readmore-more="{{ collapse.more|default('read_more')|trans({}, 'SonataAdminBundle') }}"
+                      data-readmore-less="{{ collapse.less|default('read_less')|trans({}, 'SonataAdminBundle') }}">
+            {% endif %}
+
+            {% if field_description.options.safe %}{{ value|raw }}{% else %}{{ value|nl2br }}{% endif %}
+
+            {% if field_description.options.collapse is defined %}</div>{% endif %}
```

Comment:
Repeating this condition feels weird. Maybe store the inner value in a variable and repeat it?